.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+	"singleQuote": true,
+	"tabWidth": 4,
+	"useTabs": true
```

Comment:
In what world is that a good config, change tabWidth to 2 and useTabs to false.

Reply:
@Shawak - "In the world that you cannot or will not see" is my direct answer to you. I have set Prettier to it's default opinion to avoid the kind of opinionated flame wars that personal preferences and views can spark. You can read more about their philosophy [here](https://prettier.io/docs/en/option-philosophy). You can find all the default options, specifically [tabs](https://prettier.io/docs/en/options#tab-width), [here](https://prettier.io/docs/en/options).

Reply:
No idea what page you are vising, but the default is 2 and false.

![image](https://github.com/user-attachments/assets/524dd1aa-9714-4a0d-8a29-c08ff11b8456)

Reply:
You will see the brown "Outdated" flags for code on the Github "Conversation" tab where you may still be looking.

I [have changed](https://github.com/tr1ckydev/webview-bun/pull/21/files#diff-663ade211b3a1552162de21c4031fcd16be99407aae5ceecbb491a2efc43d5d2) the `.prettierconfig` to be `{}`, ie. the default Prettier options will be observed. @tr1ckydev can set preferred project options (if any) there before merging if they so choose. 

Please step away from conflict and be aware of yourself @Shawak .